plugin/pkg/scheduler/scheduler.go

```diff
@@ -124,7 +124,9 @@ func (s *Scheduler) scheduleOne() {
 	}
 
 	go func() {
-		defer metrics.E2eSchedulingLatency.Observe(metrics.SinceInMicroseconds(start))
+		defer func() {		
```

Comment:
My bad. For some reason I thought the go routine and `start` were in a loop and thought that `start` changed before the go routine had the chance to run.